uint8 motor_id
int16 current
int16 velocity
int16 position
int16 acclX_scaled
int16 acclY_scaled
int16 acclZ_scaled
int16 gyroX_scaled
int16 gyroY_scaled
int16 gyroZ_scaled
#double acclX_scaled
#double acclY_scaled
#double acclZ_scaled
#double gyroX_scaled
#double gyroY_scaled
#double gyroZ_scaled

